modules/ROOT/partials/integrations/bunding-integrations-not-reccomended-message.adoc

```diff
@@ -0,0 +1 @@
+IMPORTANT: {companyname} does not recommend bundling `tinymce` and `{packageName}` with a module loader. Bundling these packages can be complex and error prone.
```

Comment:
The spelling of this file is wrong. It should be `bundling-integrations-not-recommended-message.adoc`

Reply:
fixed in last [commit](https://github.com/tinymce/tinymce-docs/pull/3722/commits/ef8b9a01aa0831b006df52c98a6474d3be89b2c2)